src/libs/algolia/fetchAlgolia/buildAlgoliaParameters/tests/buildOfferSearchParameters.test.ts

```diff
@@ -0,0 +1,112 @@
+import { SearchGroupNameEnumv2 } from 'api/gen'
+import { initialSearchState } from 'features/search/context/reducer'
+import { DATE_FILTER_OPTIONS, LocationType } from 'features/search/enums'
+import { LocationFilter } from 'features/search/types'
+import { buildOfferSearchParameters } from 'libs/algolia/fetchAlgolia/buildAlgoliaParameters/buildOfferSearchParameters'
+
+describe('buildOfferSearchParameters', () => {
+  const userLocation = {
+    latitude: 48.8566,
+    longitude: 2.3522,
+  }
+
+  const isUserUnderage = false
+
+  it('should return expected output', () => {
```

Comment:
`should return expected offer search parameters to build Algolia API call`

Reply:
@cedricls1 je pense qu'il y a de grande chance que juste avec ce test il y a le coverage nécessaire mais bon j'ai essayé de couvrir les cas ou les props null sont renseignés donc à voir si tu préfères que je ne laisse que le premier ça me va aussi 👍

Reply:
Du coup, tu peux les laisser :)